### PR TITLE
Add new README.md

### PR DESCRIPTION
**Pull request by soerenmartius**
This PR adds a new README that should new users to understand  Terramate better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,27 +18,31 @@
...
+Terramate CLI is an open-source Infrastructure as Code (IaC) code generation and orchestration tool for Terraform,
+OpenTofu and Terragrunt. With Terramate, you can:
 
-The main value proposition of Terramate CLI is to simplify the management and improve the scalability of your
-Infrastructure as Code projects by using a modular and efficient approach called Terramate Stacks.
+(a) **Simplify complex codebases** to make your work more productive and reliable. Break down large state files with
+stacks to reduce run-time and blast radius and keep them DRY with native code generation.
 
-## Quick Install
+(b) **Automate and orchestrate Terraform, OpenTofu and Terragrunt in any CI/CD** using Pull Request automation, GitOps
```
Comment (pedrotypes) [inline README.md:40]: Should this be the first item? It seems like more of a tangible outcome than simplifying codebases, which is more of a perk.

File: README.md [hunk 2]
```diff
@@ -52,115 +56,48 @@ With Go:
...
-🔗 **GitOps workflows that run in your existing CI/CD platform**
+## Getting Started
 
-Unlock CI/CD workflows to orchestrate your stacks in your existing CI/CD platform, such as GitHub Actions, GitLab and
-Bitbucket Pipelines. Can be used to **preview changes and plans in Pull Requests** or to **detect infrastructure drift**
-with scheduled runs.
+The fastest way to get started with Terramate is our [getting started guide](https://terramate.io/docs/cli/getting-started/).
 
-🏎 **Deploy significantly faster and more often**
+## Why simply complex codebases?
```
Comment (pedrotypes) [inline README.md:66]: Love that you cut out the part about stacks. Stacks are an important concept, but they're an implementation detail.

File: README.md [hunk 3]
```diff
@@ -52,115 +56,48 @@ With Go:
...
-👷 **Better developer productivity and experience**
+Terramate CLI enables a new approach to eliminating the root cause of these consequences: using a modular and efficient
+approach called Terramate Stacks. A stack is a combination of infrastructure code, state and configuration, which can be
+nested to split state further.
 
-Simplified management and improved productivity by providing developer-centric workflows like one-click cloning stacks
-and environments. Enable developers to use templates to deploy complex infrastructure without requiring them to
-learn the complexity of IaC.
+Organisationally, a stack is a way to establish an ownership model. By consistently sizing stacks into isolated units
+that your team members can quickly understand, assuming ownership of your infra becomes a lot simpler.
```
Comment (pedrotypes) [inline README.md:77]:
```suggestion
that your team members can quickly understand, taking control of your infra becomes a lot simpler.
```
?

File: README.md [hunk 4]
```diff
@@ -52,115 +56,48 @@ With Go:
...
+Terramate CLI supports Terraform, OpenTofu and Terragrunt, and can be run locally as well as in your existing VCS
+(Github, Gitlab) CI/CD.
 
-⚒️ **Data Sharing**
+Terramate CLI is designed to work especially well with [Terramate Cloud](https://cloud.terramate.io), which provides
+an IaC management layer and offers a generous free tier.
```
Comment (pedrotypes) [inline README.md:82]: This doesn't sound obviously great. Maybe we can list some of the benefits of the cloud here? I understand we want to keep it subtle.

**Review by pedrotypes — APPROVED** [review]
Left some comments but already looks good, I like the edits. This reads much tighter.

**Comment by soerenmartius** [comment]
> Left some comments but already looks good, I like the edits. This reads much tighter.

great feedback thank man will incorporate this!

**Pull request merged**